module/Finna/src/Finna/View/Helper/Root/OrganisationsList.php

```diff
@@ -114,15 +125,33 @@ public function __construct(
      */
     public function __invoke()
     {
-        $cacheName = 'organisations_list_' . $this->locale;
-        $list = $this->cache->getItem($cacheName);
+        return $this->generateOrganisationsList();
+    }
 
-        if (!$list) {
+    /**
+     * Generates a list of current organisations.
+     *
+     * @return array
+     */
+    protected function generateOrganisationsList(): array
+    {
+        $cacheDir = $this->cacheManager->getCache('organisation-info')->getOptions()
+            ->getCacheDir();
+        $cacheFile = "$cacheDir/organisations_list_ $this->locale.json";
+        $val = (string)(
```

Comment:
$val on vähän huono nimi muuttujalle. Nyt ei saa cachea pakotettua pois päältä, onko tarkoitus? Jos näin, niin voisi laittaa yksinkertaisemmin tyyliin: 
`$maxAge = (int)($this->organisationConfig['General']['organisationListCacheTime'] ?? null) ?: 60;`;
Ja jos 0 on validi, niin:
`$maxAge = (int)($this->organisationConfig['General']['organisationListCacheTime'] ?? null) ?? 60;`;

Reply:
laitetaan 0 validiksi